### ISSUE TITLE
In Before example, make image file names more cryptic to be a better representation of the problem

### ISSUE THREAD

**Issue by joelhsmith** [OpenAssessItToolkit/accessible_u#4]
Screen readers will read the name of an image if no alt text is present. The names are currently too nice and do not effectively demonstrate how bad it is to have missing alt text.  

Add random string or numbers at the beginning to obfuscate the file name and its purpose.
